Projects/Market/Sources/PickMarket.swift

```diff
@@ -15,7 +15,7 @@ struct PickMarket: View {
 
     var body: some View {
         hForm {
-            hSection(Market.allCases, id: \.title) { market in
+            hSection(Market.availableMarkets(), id: \.title) { market in
```

Comment:
Couldn't we use the `availableLocales` property, I think I must have missed to use that one when I reimplemented this screen in swift 😓

Reply:
the query doesn't exist anymore - thread here https://hedviginsurance.slack.com/archives/C016KUN61U3/p1643701558355619

Reply:
so we are always falling back on to hard coded values